src/tools/dartdevc.md

```diff
@@ -4,209 +4,15 @@ short-title: dartdevc
 description: A development compiler for fast, modular compilation of Dart code to JavaScript.
 ---
 
-The Dart development compiler (_dartdevc_, also known as _DDC_)
-is a Dart-to-JavaScript compiler
-that's targeted at web app development in modern browsers.
-
-dartdevc is used by tools such as the [`webdev`][] tool
-to allow you to run and debug your Dart web apps in Chrome.
+The Dart development compiler (_dartdevc_)
+compiles Dart as JavaScript AMD modules. These modules
+work for web app development in modern browsers.
 
 {{site.alert.note}}
-  dartdevc is used for _development_ only.
-  Continue to use [dart2js](/tools/dart2js)
-  through [`dart compile js`](/tools/dart-compile#js) 
-  or the [`webdev`][] tool
-  to compile for deployment.
+  Dart 2.18 removes the `dartdevc` command-line tool from the Dart
+  package, but retains the dartdevc compiler.
+  
+  Use [`dart compile js`](/tools/dart-compile#js) to compile Dart code
```

Comment:
I don't think this is true. `compile js` will produce non-debuggable JS. I think maybe we need to point to webdev instead, @sigmundch ?

Reply:
It actually looks like the removed content below might all still work, and just need to be retained?

Reply:
This all falls under the `dartdevc` command. I can excise some parts and put them elsewhere, like `webdev`. I thought you were looking for removing anything other than `dart compile js`, so I tried to stay in that lane. Can you indicate which lines I should move? All of them, changing `dartdevc` to `webdev serve`?

Reply:
My recommendation here is that we are not downplaying one command over others. Both commands are serving different purposes and each is valid for the journey the developer is at the moment (I provided additional details in the planning doc if that helps).

That means that we'd keep both references to web development and web production compilers, but change the sample commands. If a command used `dart2js`, we turn it into `dart compile js`, if a command used `dartdevc`, we turn it into `webdev serve`.

In the past, calling `dartdevc` directly was not recommended anyways, and not something I expect we had said to do in the documentation (we used the term to refer to the development compiler, but I'd be surprised if we had examples to show people how to call it directly). So I expect for the most part we used `webdev serve` to describe those use cases, and we can simply preserve good chunks of the old documentation.

Reply:
Correct - in this case we'd point them at `webdev serve`

There is still a page for webdev below, so rather than retaining the content of the dartdevc.md file, I'd recommend selecting the relevant pieces to preserve that make sense to show in the context of webdev. From looking at the contents on this file, thought, I don't think there is much I'd keep (all content about module format, etc, are not really that relevant to our developers).

Reply:
Is it a requirement that we keep old pages around? or is it OK to delete them altogether?

After looking at this more closely, I'm no longer sure I see the value in keeping around both the dartdevc.md and the dart2js.md pages. So I'd be inclined in deleting them both. If we have to keep them, then I would suggest to trim them down further. For example, by deleting anything that is describing the compilers beyond production/development (e.g. I'd suggest deletion the whole explanation of _modules_ and _AMD_).

One suggestion:
* _"Version note: Since Dart 2.18 the `dart2js` web production compiler is only available from `dart compile js`(link)"_, and
* _"Version note: Since Dart 2.18 the `dartdevc` web development compiler is only available from `webdev serve`(link)."_ 

Another suggestion (I slightly prefer this one):
* _"dart2js is Dart's web production compiler. Version note: Since Dart 2.18 the `dart2js` compiler can only be invoked from `dart compile js`. It is also used internally by deployment tools like `webdev build`"_, and
* _"dartdevc is Dart's web development compiler. Version note: Since Dart 2.18 the `dartdevc` compiler cannot be invoked directly, but it is used internally by development tools like  `webdev serve`."_